stdlib/public/core/Result.swift

```diff
@@ -58,8 +58,8 @@ public enum Result<Success, Failure: Error> {
   /// the error value of a result by wrapping it in a custom `Error` type:
   ///
   ///     struct DatedError: Error {
-  ///         var error: Error
-  ///         var date: Date
+  ///         let error: Error
+  ///         let date: Date
```

Comment:
This change doesn't seem necessary to me — it's common practice to have struct properties declared with `var`.

Reply:
Agreed. I reverted those properties!
Thank you for reviewing my commit.